src/issues.ts

```diff
@@ -0,0 +1,73 @@
+import {get} from "./helpers/github";
```

Comment:
That's really good unfortunately  I don't think we can rely on github's api, because the rate limits it gives us. I recommend we get data from the [osscameroon_issues.json](https://github.com/osscameroon/miniyotas/blob/main/src/res/osscameroon_issues.json) file. I saw you have used data like the language and the repo on this PR, but I guess by updating the script output [here](https://github.com/osscameroon/miniyotas/blob/a4c65cb58657f4f0be9c1a4a32a7ff5c502272a3/.github/workflows/update_osscameroon_issues/get_opened_issues.sh#L17) we can probably get you the data you need for your output.

I suggest you remove the request to the `github` api and make use of this file, in can you need specific fields from that file create an issue, assign it to me and  I will update the script for you (or you can do it yourself :) ).

Reply:
Thanks for the feedback.
So I can remove the projects page and directly present the issues by getting the data from the osscameroon_issues.json file ?